.github/dependabot.yaml

```diff
@@ -1,5 +1,13 @@
 version: 2
 updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily" 
+    target-branch: "main"
+    versioning-strategy: "lockfile-only"
```

Comment:
This is not available for Go, so it doesn't have any effect:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy